pkg/util/integer/integer.go

```diff
@@ -43,3 +45,16 @@ func Int64Min(a, b int64) int64 {
 	}
 	return a
 }
+
+func RoundToInt(a float64) int {
```

Comment:
Much simpler, thanks! My head was spinning around at the time I was writting this helper.